sample-apps/SampleAppSpec.md

```diff
@@ -55,7 +55,7 @@ This configuration file should be set by an environment variable (SAMPLE_APP_CON
 ```
 ---
 Host: "0.0.0.0"                       # Host - String Address
-Port: "8080"                          # Port - String Port
+Port: "4567"                          # Port - String Port
```

Comment:
Why is this change required? From my understanding the only sample app that leverages this port is Java? Should java be changed instead?

Reply:
Also, the port is configurable so isn't this merely an example?

Reply:
I thought it would be easier to change the sample app listening ports to default to 4567 since that is the default sample app listening port in the test framework.  Although, this port is configurable so the user could always change it and it only really serves as a starting point.  However, when I was also testing on ecs, there was a port conflict with using 8080 and so I thought following the behavior of the Java app would be better as a standard unless there's a specific reason we should use 8080 instead?

Reply:
I took a look at the rest of the sample apps and I believe that only the `java` one does not implement a `config.yaml`. All of the sample apps have a configuration file present. In this case I think it's find to change the default to match the java sample app.